### ISSUE TITLE
Docker cannot start

### ISSUE THREAD

**Issue by david50407** [docker/for-mac#1079]
Diagnostic ID: `6F447B31-9EE8-4206-B878-409352503ABA`

```
Docker for Mac: version: 1.13.0-rc4-beta34.1 (b4ff3714d)
OS X: version 10.12.1 (build: 16B2657)
logs: /tmp/6F447B31-9EE8-4206-B878-409352503ABA/20161223-234112.tar.gz
failure: com.docker.slirp is not running
[OK]     vmnetd
[OK]     dns
[ERROR]  driver.amd64-linux
         com.docker.driver.amd64-linux -db is not running
[OK]     virtualization VT-X
[OK]     app
[ERROR]  moby
         /Users/Davy/Library/Containers/com.docker.docker/Data/com.docker.driver.amd64-linux/console-ring does not exist
[OK]     system
[OK]     moby-syslog
[ERROR]  db
         Connection refused (ECONNREFUSED) connecting to /Users/Davy/Library/Containers/com.docker.docker/Data/s40: check if service is running
         com.docker.db is not running
[OK]     env
[OK]     virtualization kern.hv_support
[ERROR]  slirp
         Unexpected error ((Failure
  "Error connecting socket to 9p endpoint unix:/Users/Davy/Library/Containers/com.docker.docker/Data/s51: Unix.Unix_error(Unix.ECONNREFUSED, \"connect\", \"\")")) connecting to /Users/Davy/Library/Containers/com.docker.docker/Data/s51
         com.docker.slirp is not running
         slirp check failed with: (Failure
  "Error connecting socket to 9p endpoint unix:/Users/Davy/Library/Containers/com.docker.docker/Data/s52: Unix.Unix_error(Unix.ECONNREFUSED, \"connect\", \"\")")
[ERROR]  osxfs
         com.docker.osxfs is not running
[OK]     moby-console
[OK]     logs
[ERROR]  docker-cli
         Connection refused (ECONNREFUSED) connecting to /var/run/docker.sock: check if service is running
         Connection refused (ECONNREFUSED) connecting to /Users/Davy/Library/Containers/com.docker.docker/Data/s60: check if service is running
         docker ps failed
[OK]     menubar
[OK]     disk
```

log: [20161223-234112.tar.gz](https://github.com/docker/for-mac/files/671044/20161223-234112.tar.gz)

**Comment by djs55** [comment]
It looks like this is related to #1062

In beta 34 we added extra integrity checks to look for problems in the `Docker.qcow2` before starting the VM. I see in your logs:
```
block device raised exception: (Failure "Found two references to cluster 46685: 3522.22528 and 3522.22272")
```
This means that two 64KiB regions of the disk (clusters) are mapped to the same physical part of the file, so that one write will also overwrite the other. It would be unsafe to write to the disk in this state. I believe this happened in the past -- possibly around beta 33 -- but has only been noticed after the beta 34 upgrade.

I think the only solution is to reset to factory defaults or delete the file `~/Library/Containers/com.docker.docker/Data/com.docker.driver.amd64-linux/Docker.qcow2` and restart the app. Unfortunately your images and containers will need to be rebuilt -- I'm sorry about this. I think this should allow the app to start and to work properly.

I'll close this ticket since I believe the corruption was introduced in an earlier version (probably by running too many `hyperkit` processes). However if it happens again after the reset, please do open a new issue.

I apologise for any inconvenience caused.
